hack/common.sh

```diff
@@ -50,7 +50,7 @@ ROOK_CSV="$OUTDIR_TEMPLATES/rook-csv.yaml.in"
 OCS_CSV="$OUTDIR_TEMPLATES/ocs-operator.csv.yaml.in"
 FCS_CSV="$OUTDIR_TEMPLATES/ics-operator.csv.yaml.in"
 
-LATEST_ROOK_IMAGE="rook/ceph:v1.10.0"
+LATEST_ROOK_IMAGE="subham03/csv:2"
```

Comment:
```suggestion
LATEST_ROOK_IMAGE="subham03/csv:2"
```
need to use official tag

Reply:
reverted, good to go